docs/messaging/kafka.md

```diff
@@ -159,4 +158,20 @@ This section defines how to apply semantic conventions when collecting Kafka met
 | messaging.kafka.consumer.lag_sum              | Gauge         | Int64      | lag sum | `{lag sum}` | Current approximate sum of consumer group lag across all partitions of topic | `group` | The ID (string) of a consumer group |
 |                                               |               |            |         |             |                                                                              | `topic` | The ID (integer) of a topic |
 
+### Broker Metrics
+
+**Description:** Kafka Broker level metrics.
+
+| Name                                         | Instrument    | Value type | Unit   | Unit ([UCUM](/docs/general/metrics.md#instrument-units)) | Description    | Attribute Key | Attribute Values |
+| ---------------------------------------------| ------------- | ---------- | ------ | -------------------------------------------- | -------------- | ------------- | ---------------- |
+| messaging.kafka.brokers.count                | UpDownCounter | Int64      | brokers   | `{broker}`   | sum of brokers in the cluster | | |
+| messaging.kafka.brokers.consumer.fetch.rate  | Gauge         | Double     | fetches per second | `{fetch}/s`  | Average consumer fetch Rate. | `state` | `in`, `out` |
+| messaging.kafka.brokers.network.io           | Counter       | Int64      | bytes   | `By`     |  The bytes received or sent by the broker. | | |
+| messaging.kafka.brokers.requests.latency     | Gauge         | Double     | ms        | `{ms}` | Average Request latency in ms. | | |
+| messaging.kafka.brokers.requests.rate        | Gauge         | Double     | requests per second | `{request}/s`| Average request rate per second. | | |
+| messaging.kafka.brokers.requsts.size         | Gauge         | Double     | bytes     | `By`         | Average request size in bytes. | | |
```

Comment:
this would probably be best represented with histogram to allow distribution and then a rate counter is not needed.
Another possibility would be to only report `messaging.kafka.brokers.network.io` with direction attribute or use  `messaging.kafka.brokers.request.bytes` counter counting all the bytes .

Reply:
Hi @lmolkova , is there an example of a histogram metric available and how that would be defined?

Reply:
you can find examples throughout this repo, for example, take a look at `http.client.request.duration`. there is a lot of information in the [spec repo](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics) and also take a look at metric docs on [opentelemetry.io](https://opentelemetry.io/docs/specs/otel/metrics/data-model/#histogram)